editor/themes/editor_theme_manager.cpp

```diff
@@ -396,8 +396,8 @@ void EditorThemeManager::_create_shared_styles(const Ref<EditorTheme> &p_theme,
 		if (!p_config.dark_theme) {
 			// Darken some colors to be readable on a light background.
 			p_config.success_color = p_config.success_color.lerp(p_config.mono_color, 0.35);
-			p_config.warning_color = p_config.warning_color.lerp(p_config.mono_color, 0.35);
-			p_config.error_color = p_config.error_color.lerp(p_config.mono_color, 0.25);
+			p_config.warning_color = Color("#d18f19");
+			p_config.error_color = Color("#cd3838");
```

Comment:
I suggest using the `Color(float, float, float)` form which avoids storing a string in the binary, and therefore reduces binary size ever so slightly. I did this on all colors in the codebase a while ago and it reduced binary size by a non-negligible amount.

`Color::hex(0xrrggbbaa)` might reduce binary size further by using 1 integer instead of 3 floats, but that should be evaluated later.

Reply:
`Color::hex(0xrrggbbaa)` can be shortened to `Color(0xrrggbbaa)`, no? It's a direct replacement for hex string.

Reply:
My assumption was that the compiler was smarter than this but I suppose that these are among the many things to keep in mind. I did not expect for the Colors to be basically generated on the spot even for constant Strings.

Reply:
I went with the 3-float approach then, but I agree reducing it in the future to a single hex int sounds interesting, especially for iterating on the colors and not having to constantly convert them to a [0, 1] float range.

Reply:
> By the way, I may be wrong but I believe the bits are reversed in pure hex: 0xaabbggrr.

`ARGB` was used in Godot 3.x, but Godot 4.0 switched to `RGBA`.